src/Hooks.res

```diff
@@ -71,3 +71,20 @@ module ArrayField = {
   @module("react-hook-form")
   external use: (~option: option) => t<'data> = "useFieldArray"
 }
+
+module WatchValues = {
+  type t
```

Comment:
sorry @gaku-sei -san, I am still a bit noob here, do you know how to make it type safe here

for example:
If I use sth like this
```
let firstName = Hooks.WatchValues.use(
      ~option=Hooks.WatchValues.option(~control, ~name="firstName", ()),
    )
```
I expect that firstName should be a String